src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -194,6 +195,9 @@ def load_arguments(self, _):
             c.argument('python_version', help='The version used to run your web app if using Python, e.g., 2.7, 3.4')
             c.argument('net_framework_version', help="The version used to run your web app if using .NET Framework, e.g., 'v4.0' for .NET 4.6 and 'v3.0' for .NET 3.5")
             c.argument('linux_fx_version', help="The runtime stack used for your linux-based webapp, e.g., \"RUBY|2.3\", \"NODE|6.6\", \"PHP|5.6\", \"DOTNETCORE|1.1.0\". See https://aka.ms/linux-stacks for more info.")
+            c.argument('windows_fx_version', help="[Preview] Windows Containers web app only. The docker image name used for your windows container webapp, e.g., microsoft/nanoserver:ltsc2016")
```

Comment:
Suggest remove some redundant wordings: `[Preview] a docker image name used for your Windows container based webapp e.g., microsoft/nanoserver:ltsc2016"`